examples/neural_compressor/optical-character-recognition/README.md

```diff
@@ -0,0 +1,50 @@
+<!---
+Copyright 2020 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Optical Character Recognition
+
+The script [`run_ocr.py`](https://github.com/huggingface/optimum-intel/blob/main/examples/neural_compressor/optical-character-recognition/run_ocr.py)
+allows us to apply different quantization approaches (such as dynamic and static quantization) 
+using the [Intel Neural Compressor ](https://github.com/intel/neural-compressor) library for optical character recognition tasks and [IAM](https://fki.tic.heia-fr.ch/databases/iam-handwriting-database) datasets.
+
+Note that this case is from https://github.com/NielsRogge/Transformers-Tutorials/blob/master/TrOCR/Evaluating_TrOCR_base_handwritten_on_the_IAM_test_set.ipynb, And it only support TrOCR models.
+
+## Prepare datasets
+```bash
+wget https://layoutlm.blob.core.windows.net/trocr/dataset/IAM.tar.gz
+tar xvf IAM.tar.gz
+```
+
+## Run Command
+The following example applies post-training static quantization on TrOCR small fine-tuned on the IAM dataset.
+```bash
+python run_ocr.py \
+    --model_name_or_path microsoft/trocr-small-handwritten \
+    --datasets_dir IAM \
+    --apply_quantization \
+    --quantization_approach static \
+    --max_eval_samples 1000 \
```

Comment:
Would it be reasonable to reduce the number of `max_eval_samples` ? (to something like 500), otherwise evaluation can take quite some time

Reply:
Had set to 500.